metadata-ingestion/src/datahub/utilities/hive_schema_to_avro.py

```diff
@@ -28,6 +28,7 @@ class HiveColumnToAvroConverter:
         "bigint": "long",
         "varchar": "string",
         "char": "string",
+        "unknown": "null",
```

Comment:
null is already the fallback https://github.com/datahub-project/datahub/blob/5c40390a923a2a44f23290e4f1a2168820993fca/metadata-ingestion/src/datahub/utilities/hive_schema_to_avro.py#L191

```suggestion
```